torch/_prims/nvfuser_executor.py

```diff
@@ -237,6 +237,10 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
                 )
                 is not None
             )
+        if "getitem" in node.name:
+            # Check if the node unpacks a tuple from a supported node
+            node_to_unpack = node.args[0]
+            return self.is_node_supported(submodules, node_to_unpack)
```

Comment:
I'm slightly leaning towards clean up `getitem` at the beginning of the graph in partitioner, instead of having more complicated logic in op support query. For now it works fine, since partitioner goes from consumer to producer and we don't really fuse anything as we propose partition group. But that implementation could change :frowning:

Reply:
It's of course a matter of taste. I think special casing `getitem` and processing it differently may cause bugs. There are two situations with getitem it's either unpack a node that's supported or not, and we can't unconditionally either reject or accept getitem. Previously we have already relied on the partitioner (always rejecting getitem) to do the right thing and it didn't.